data/extension/init/templates/tinygo/envoy.filters.http/default/main.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
Can we include extension configuration into this init template ?

Reply:
What is the resolution ?

I don't mind what configuration format is.

I'd like to see how `httpContext` instances will get a hold of that configuration.

Reply:
same: https://github.com/tetratelabs/getenvoy/pull/104#discussion_r589904082

will work on reading configurations.